src/gui/properties/propertieswidget.h

```diff
@@ -55,15 +55,18 @@ class QAction;
 class QTimer;
 QT_END_NAMESPACE
 
-class PropertiesWidget: public QWidget, private Ui::PropertiesWidget
+class PropertiesWidget : public QWidget, private Ui::PropertiesWidget
```

Comment:
I saw in another PR that uncrustify did the opposite of this
Should i undo these?